docs_source_files/content/front_matter/copyright_and_attributions.ja.md

```diff
@@ -52,215 +52,105 @@ weight: 1
-      defend, and hold each Contributor harmless for any liability
-      incurred by, or claims asserted against, such Contributor by reason
-      of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
-
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
+   使用、複製、および頒布に関する条項
+
+   1. 定義
+
+      「ライセンス」とは、このドキュメントの第1項から第9項までで定義している、使用、複製、および頒布に関する条項を指します。
+
+      「ライセンサー」とは、著作権所有者、あるいは著作権所有者がライセンス付与対象として認めた者を指します。
+
+      「法人」とは、行為者と、行為者を管理するか行為者により管理されるか行為者共通の管理下にある他のすべての者とから成る連合体を指します。
+      この定義における「管理」とは、(i) 契約またはその他により、直接または間接的にこの法人の指揮・経営を行う権限、または (ii) この法人の50%以上の株式の所有権または (iii) 受益所有権を有することを指します。
+
+      「あなた」とは、本ライセンスにより付与される権利を行使する個人または法人を指します。
+
+      「ソース」形式とは、ソフトウェアのソースコード、ドキュメントソース、設定ファイルといった、変更を加えるのに好都合な形式を指します。
+
+      「オブジェクト」形式とは、コンパイルされたオブジェクトコード、生成されたドキュメント、他のメディアへの変換物といった、ソース形式の機械的な変換により生じる形式を指します。
+
+      「成果物」とは、ソース形式であるとオブジェクト形式であるとを問わず、製作物に挿入または添付される（後出の付録に例がある）著作権表示で示された著作物で、本ライセンスに基づいて利用が許されるものを指します。
+
+      「派生成果物」とは、編集上の改訂、注解、推敲など、成果物を基にしていて全体としてオリジナル著作物と呼べるような製作物全般を指します。
+      本ライセンスでは、成果物や派生成果物から分離できる製作物や、成果物や派生成果物のインタフェースへの単なるリンク（または名前によるバインド）を、派生成果物に含めません。
+
+      「コントリビューション」とは、成果物のオリジナルバージョンならびに成果物または派生成果物への変更や追加も含めて、著作権所有者あるいは著作権所有者が認めた個人または法人による成果物への組み込みを意図してライセンサーに提出される著作物全般を指します。
+      この定義における「提出」とは、成果物を論じたり改良するためにライセンサーまたはその代理者により管理される電子的メーリングリスト、ソースコード管理システム、問題追跡システムといった、電子的方法、口頭、または書面で、ライセンサーまたはその代理者に情報を送ることを指します。
+      ただし、著作権所有者が書面で「コントリビューションでない」と明示したものは除きます。
+
+      「コントリビューター」とは、ライセンサーおよびその代理を務める個人または法人で、自分のコントリビューションがライセンサーに受領されて成果物に組み込まれた者を指します。
+
+   2. 著作権ライセンスの付与
+      本ライセンスの条項に従って、各コントリビューターはあなたに対し、ソース形式であれオブジェクト形式であれ、成果物および派生成果物を複製したり、派生成果物を作成したり、公に表示したり、公に実行したり、サブライセンスしたり、頒布したりする、無期限で世界規模で非独占的で使用料無料で取り消し不能な著作権ライセンスを付与します。
+
+   3. 特許ライセンスの付与
+      本ライセンスの条項に従って、各コントリビューターはあなたに対し、成果物を作成したり、使用したり、販売したり、販売用に提供したり、インポートしたり、その他の方法で移転したりする、無期限で世界規模で非独占的で使用料無料で取り消し不能な（この項で明記したものは除く）特許ライセンスを付与します。
+      ただし、このようなライセンスは、コントリビューターによってライセンス可能な特許申請のうち、当該コントリビューターのコントリビューションを単独または該当する成果物と組み合わせて用いることで必然的に侵害されるものにのみ適用されます。
+      あなたが誰かに対し、交差請求や反訴を含めて、成果物あるいは成果物に組み込まれたコントリビューションが直接または間接的な特許侵害に当たるとして特許訴訟を起こした場合、本ライセンスに基づいてあなたに付与された特許ライセンスは、そうした訴訟が正式に起こされた時点で終了するものとします。
```

Comment:
Hi @kzhirata ,

Thanks for the PR. Can you minimize the line width with a max of 70-80 characters.

Thanks,
Harsha

Reply:
@Harsha509 Thank you for the review. 
I've add some line breaks.